modules/apps/frontend-editor/frontend-editor-ckeditor-web/node-scripts.config.js

```diff
@@ -15,20 +15,27 @@ module.exports = {
 			'Alignment',
 			'BlockQuote',
 			'Bold',
+			'ButtonView',
 			'ClassicEditor',
 			'EditorConfig',
 			'Essentials',
 			'Font',
 			'GeneralHtmlSupport',
 			'Heading',
 			'HorizontalLine',
+			'Image',
+			'ImageBlock',
+			'ImageCaption',
+			'ImageInline',
+			'ImageStyle',
+			'ImageToolbar',
```

Comment:
These plugins add image context toolbar features. They are not strictly in the scope of the task, but I found them very nice. They are a quick win to add.